assembl/static2/js/app/components/login/signupForm.jsx

```diff
@@ -93,17 +91,17 @@ class SignupForm extends React.Component<SignupFormProps, SignupFormState> {
     }
   }
 
-  toggleCheck() {
-    this.setState({ checked: !this.state.checked });
-  }
+  toggleCheck = (termFormType) => {
+    this.setState({ [termFormType]: !this.state[termFormType] });
```

Comment:
For forward compatibility reason, you should use the new syntax:
`this.setState((prevState) => ({ [termFormType]: !prevState[termFormType] }));`

https://reactjs.org/docs/react-component.html#setstate